tests/Doctrine/Tests/DBAL/Functional/ParameterTypes/AsciiTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\ParameterTypes;
+
+use Doctrine\DBAL\Driver\AbstractSQLServerDriver;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+class AsciiTest extends DbalFunctionalTestCase
+{
+    public function testAsciiBinding(): void
+    {
+        if (! $this->connection->getDriver() instanceof AbstractSQLServerDriver) {
+            self::markTestSkipped('Driver does not support ascii string binding');
+        }
+
+        $statement = $this->connection->prepare('SELECT sql_variant_property(?, \'BaseType\')');
+
+        $statement->bindValue(1, 'test', ParameterType::ASCII);
+        $statement->execute();
```

Comment:
This had to be reworked during the rebase because in `2.x` `Statement::execute()` doesn't return a result.

Reply:
Looks good thanks!